src/shared/lib/oql/oqlfilter.js

```diff
@@ -1,4 +1,4 @@
-// Requires parser to be window object, window.oql_parser
+// Requires parser to be window object, oql_parser
```

Comment:
I still find this comment a bit confusing. Does it actually require anything from the window object, or does our tooling (Webpack, maybe helped by Babel?) just make the import statement below work?

Reply:
no this is not related to tooling.  it's basically just saying, this is an expected global

Reply:
actually this comment is defunct, right? b/c we now import it below.  strike comment?

Reply:
heh this is the result of a find and replace, this comment should be deleted. good catch